README.md

```diff
@@ -1,17 +1,56 @@
-go-libipfs
-=======================
+<h1 align="center">
+go-libipfs 🍌
```

Comment:
Maybe this is repeating too many bananas 😃

Reply:
I wanted to leave a copy-pastable unicode banana there too, the big one is not copy-pastable (and github won't let me make text that big).  This was also inspired by Downshift's README https://github.com/downshift-js/downshift/blob/master/README.md